src/plugins/explore/public/components/visualizations/scatter/scatter_exclusive_vis_options.tsx

```diff
@@ -27,6 +28,15 @@ export const ScatterExclusiveVisOptions = ({ styles, onChange }: ScatterVisOptio
     });
   };
 
+  const [pointAngle, handlePointAngle] = useDebouncedNumericValue(
+    styles.angle,
+    (val) => onChange({ ...styles, angle: val }),
+    {
+      min: 0,
```

Comment:
i see we define min max to the style accordian as well so if we had to change the min and max (even though unlikely) we will have to update it in two places.

wouldn't it be easier to have a higher level constants file or types or some factory, essentially a single place where a developer can see defined vis configs. and then within the component options.  we are just accessing the configs? 

that way we define things once and as a developer i can just look in one place to configure these values

Reply:
@kavilla the `min` and `max` here are the parameter of `useDebouncedNumericValue`, not default vis styles